### PR TITLE
Add more logging

### PR DESCRIPTION
**Pull request by gemmai95**
Log out details of receipt request to rrm. This was necessary to get more details in any possible failovers.

## Checklist
  - [x] CHANGELOG.md updated? (if required)

### CODE REVIEW

File: app/response_processor.py [hunk 1]
```diff
@@ -122,6 +122,7 @@ def _send_receipt(self, decrypted, xml):
 
             response = session.post(endpoint, data=xml, headers=headers, verify=False, auth=auth)
             self.logger = self.logger.bind(status=response.status_code)
+            self.logger.info("Request details", data=xml, header=headers)
```
Comment (insacuri) [inline app/response_processor.py]: What info is contained in the xml and the headers?  And how big are they?  We don't want huge globs of data in the logs if we can avoid it.  I'm worried about possibly putting something sensitive in the logs

Comment (warrenbailey) [reply app/response_processor.py]: I don't think they'll be anything sensitive in the receipt, its just the respondent id and the tx_id. Though I agree we shouldn't log all the xml and headers, just the useful bits. 

Maybe just log the respondent id when it's receipted

**Review by warrenbailey — CHANGES_REQUESTED** [review]

File: app/response_processor.py (hunk 1 above)
Comment (JamesGardiner) [reply app/response_processor.py]: What about the request url - is that logged anywhere, as that would probably be quite valuable.

Comment (gemmai95) [reply app/response_processor.py]: That is already bound to the logger

**Comment by gemmai95** [comment]
[Add more logging to services that receipt to RRM (2)](https://trello.com/c/oBKmMLMu/412-add-more-logging-to-services-that-receipt-to-rrm-2)

File: app/response_processor.py (hunk 1 above)
Comment (warrenbailey) [reply app/response_processor.py]: If the request url and tx_id are already logged then I'm not sure this PR is needed.

File: app/response_processor.py [hunk 2]
```diff
@@ -125,7 +125,10 @@ def _send_receipt(self, decrypted, xml):
 
             try:
                 response.raise_for_status()
-                self.logger.info("Sent receipt")
+                namespace = {'receipt': 'http://ns.ons.gov.uk/namespaces/resources/receipt'}
+                tree = etree.fromstring(xml)
+                respondent_id = tree.find('receipt:respondent_id', namespace).text
+                self.logger.info("Sent receipt for ", respondent_id=respondent_id)
```
Comment (warrenbailey) [inline app/response_processor.py]: It needs some XML exception handling here.

Though if it fails we still need to send the receipt

**Review by warrenbailey — CHANGES_REQUESTED** [review]

**Review by warrenbailey — APPROVED** [review]

**Pull request merged**